docs/migration.md

```diff
@@ -40,9 +40,11 @@ license: |
 
 - Since 0.5.0, Celeborn deprecate `celeborn.client.spark.shuffle.forceFallback.enabled`. Please use `celeborn.client.spark.shuffle.fallback.policy` instead.
 
-- Since 0.5.0, Celeborn master REST API `/exclude` uses POST method and the parameters `add` and `remove` use form parameters instead.
+- Since 0.5.0, Celeborn master REST API `/exclude` uses POST method and the parameters `add` and `remove` use form parameters instead, and use media type `application/x-www-form-urlencoded` instead of `text/plain`.
```

Comment:
IMO, you should move this description of migration to `## Upgrading from 0.5 to 0.6` or `## Upgrading from 0.5.0 to 0.5.1`.

Reply:
currently, with text/plain， the request will fail.
This pr does not introduce any user face change.

Reply:
with text/plain media type, the form params can not be transferred to server side and were always empty.

so, this change was from 0.5.0

Reply:
updated the pr title and add `annotations` keyword.

Reply:
0.5.0 is already released..

Reply:
@turboFei, although the change was from 0.5.0, this followup would be released in 0.5.1 or 0.6.0. Therefore, you should not update the migration between 0.4 and 0.5.

Reply:
ok, updated